tutorials/intermediate/constrainedlayout_guide.py

```diff
@@ -430,6 +450,36 @@ def docomplicated(suptitle=None):
 example_plot(axs[0], fontsize=12)
 axs[1].set_position([0.2, 0.2, 0.4, 0.4])
 
+###############################################################################
+# .. _compressed_layout:
+#
+# Grids of fixed aspect-ratio Axes: "compressed" layout
+# =====================================================
+#
+#
+# By default, ``constrained_layout`` acts on the original position of an axes.
+# However, when Axes have fixed aspect ratios, one side is made shorter.  For
+# a grid of such Axes, this can leave a large gap in the shortened direction.
+# In the following the Axes are square, but the figure quite wide so there is
+# a horizontal gap:
```

Comment:
Is "original position of an axes" a concept we have? Maybe

 ```suggestion
# By default, ``constrained_layout`` acts on the grid spaces reserved for the
# Axes. However, when Axes have fixed aspect ratios, one of their sides is made
# shorter. so that it fits the reserved space while ensuring the fixed aspect
# ratio. A grid of such Axes, may thus have large gaps between the axes:
```